decidim-core/lib/decidim/api/interfaces/component_interface.rb

```diff
@@ -6,15 +6,15 @@ module ComponentInterface
       include Decidim::Api::Types::BaseInterface
       description "This interface is implemented by all components that belong into a Participatory Space"
 
-      field :id, ID, "The Component's unique ID", null: false
-
-      field :name, TranslatedFieldType, "The name of this component.", null: false
-
-      field :weight, Integer, "The weight of the component", null: false
-
-      field :participatory_space, ParticipatorySpaceType, "The participatory space in which this component belongs to.", null: false
-
-      field :url, String, "The URL of this component.", null: false
+      implements Decidim::Core::TimestampsInterface
+
+      field :id, ID, "The unique id of this component", null: false
+      field :name, TranslatedFieldType, "The name of this component", null: false
+      field :participatory_space, ParticipatorySpaceType, "The participatory space of this component", null: false
+      field :published_at, Decidim::Core::DateTimeType, "The publication timestamp of this component", null: false
```

Comment:
Consistency with `TimestampsInterface` 

```suggestion
      field :published_at, Decidim::Core::DateTimeType, "The date and time this object was published", null: false
```